index.html

```diff
@@ -0,0 +1,157 @@
+      <p class="card-works-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="card-works-tech">
+        <li class="card-works-tech-items">html</li>
+        <li class="card-works-tech-items">bootstrap</li>
+        <li class="card-works-tech-items">Ruby</li>
+      </ul>
+      <button class="card-works-button" type="button">See Project</button>
+    </div>
+    <div class="card-works">
+      <h2 class="card-works-title">Profesional Art Printing Data</h2>
+      <p class="card-works-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="card-works-tech">
+        <li class="card-works-tech-items">html</li>
+        <li class="card-works-tech-items">bootstrap</li>
+        <li class="card-works-tech-items">Ruby</li>
+      </ul>
+      <button class="card-works-button" type="button">See Project</button>
+    </div>
+    <div class="card-works">
+      <h2 class="card-works-title">Profesional Art Printing Data</h2>
+      <p class="card-works-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="card-works-tech">
+        <li class="card-works-tech-items">html</li>
+        <li class="card-works-tech-items">bootstrap</li>
+        <li class="card-works-tech-items">Ruby</li>
+      </ul>
+      <button class="card-works-button" type="button">See Project</button>
+    </div>
+    <div class="card-works">
+      <h2 class="card-works-title">Profesional Art Printing Data</h2>
+      <p class="card-works-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="card-works-tech">
+        <li class="card-works-tech-items">html</li>
+        <li class="card-works-tech-items">bootstrap</li>
+        <li class="card-works-tech-items">Ruby</li>
+      </ul>
+      <button class="card-works-button" type="button">See Project</button>
+    </div>
+<!--card works section end-->
+  </div>
+  <div class="container-3">
+    <div class="about-me">
+      <h1 class="about-me-title">About me</h1>
+      <p class="about-me-description">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      <button class="about-me-button" type="button">Get My Resume</button>
+    </div>
+    <div class="skills">
+      <ul>
+        <li class="language">Language
+          <ul>
+            <li class="skill">Javascript</li>
+            <li class="skill">Ruby</li>
+            <li class="skill">Html</li>
+            <li class="skill">Css</li>
+          </ul>
+        </li>

```

Comment:
- [ ] Great job coming up with a proper markup for the Languages section. It however does not look like the provided Figma design as it is aligned slightly yo the right and it is required to be aligned at the middle. Adding `align-items: center;` should fix this.

**This is how your design currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/176833208-b86ba6fb-c8e7-47d0-aa66-052357ca7b40.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/176833322-8dcd0e9a-5c1f-4e3f-91b1-0426e46dee30.png)

**Kindly make sure that you do the same for the skills section as well**